### PR TITLE
Remove implicit VL according #60

### PR DESCRIPTION
**Pull request by kito-cheng**
This is the corresponding changes for #60.

Changes:
- Remove `rvv_intrinsic_funcs_vl` and `rvv_intrinsic_funcs_vl.md`
- Update `rvv_intrinsic_funcs` and `rvv_intrinsic_funcs.md`
- Add new_vl argument to vlseg*e*ff and vle*ff
- Update example code
- Remove vreadvl
- Update doc for vl arg

### CODE REVIEW

File: rvv-intrinsic-rfc.md [hunk 1]
```diff
@@ -246,6 +249,30 @@ size_t vsetvlmax_e8m8 ();
 
 There is no need to specify the behavior of tail and masked-off elements being undisturbed or agnostic. The default setting is tail agnostic and masked-off undisturbed. If users do not want to keep the values in masked-off elements, they could pass `vundefined()` as the `maskedoff` value.
 
+`SEW` and `LMUL` are a part of the naming. They are static information for the
+intrinsics.
+
+All of the intrinsic function having a `vl` argument to specify the active
+vector length, except few functions are work with fixed length, e.g. `vmv.x.s`,
+`vfmv.f.s` and whole register move/load/store.
```
Comment (zakk0610) [inline rvv-intrinsic-rfc.md:257]: 1. IIRC,  whole register intrinsic function are not supported, shuld we mention them?
2. Could you list all vl-less intrinsic functions? I had list all in https://github.com/riscv/rvv-intrinsic-doc/issues/42 (If I did not miss something..)

Comment (kito-cheng) [reply rvv-intrinsic-rfc.md]: Thanks! updated the list.

File: rvv-intrinsic-rfc.md [hunk 2]
```diff
@@ -226,14 +225,18 @@ float64:
...
 | **8**        | N/A         | N/A             | N/A             | vfloat64m1x8_t | N/A            | N/A            | N/A
 
-## Configuration-Setting<a name="configuration-setting"></a>
+## Configuration-Setting and `vl` Argument<a name="configuration-setting"></a>
 
-`SEW` and `LMUL` are a part of the naming. They are static information for the intrinsics.
-There are two variants of configuration setting intrinsics. `vsetvl` is used to set `vl` according to the given AVL. `vsetvlmax` is used to set `vl` to VLMAX.
+There are two variants of configuration setting intrinsics. `vsetvl` is used to
+get the active vector length (`vl`) according to the given application vector
+length (`AVL`), `vsetvlmax` is used to get the maximal active vector length
```
Comment (Hsiangkai) [inline rvv-intrinsic-rfc.md:232]: "according to the given application vector length(`AVL`), `SEW` and `LMUL`."

File: rvv-intrinsic-rfc.md [hunk 3]
```diff
@@ -246,6 +249,31 @@ size_t vsetvlmax_e8m8 ();
 
 There is no need to specify the behavior of tail and masked-off elements being undisturbed or agnostic. The default setting is tail agnostic and masked-off undisturbed. If users do not want to keep the values in masked-off elements, they could pass `vundefined()` as the `maskedoff` value.
 
+`SEW` and `LMUL` are a part of the naming. They are static information for the
+intrinsics.
+
+All of the intrinsic function having a `vl` argument to specify the active
```
Comment (Hsiangkai) [inline rvv-intrinsic-rfc.md:255]: functions have

File: rvv-intrinsic-rfc.md [hunk 4]
```diff
@@ -246,6 +249,31 @@ size_t vsetvlmax_e8m8 ();
...
+vector length, except few functions are work with fixed length, e.g. `vmv.x.s`,
```
Comment (Hsiangkai) [inline rvv-intrinsic-rfc.md:256]: except a few functions operate regardless of `vl`.

File: rvv-intrinsic-rfc.md [hunk 5]
```diff
@@ -246,6 +249,31 @@ size_t vsetvlmax_e8m8 ();
...
+`vfmv.f.s`, `vundefined`, `vreinterpret`, `vlmul_ext`, `vlmul_trunc`, `vget`,
+`vset` and `vcreate`.
+
+The intrinsic function will only operate at most `VLMAX` element if `vl`
```
Comment (Hsiangkai) [inline rvv-intrinsic-rfc.md:260]: The intrinsic functions

File: rvv-intrinsic-rfc.md [hunk 6]
```diff
@@ -246,6 +249,31 @@ size_t vsetvlmax_e8m8 ();
...
+argument are larger than `VLMAX`.
```
Comment (Hsiangkai) [inline rvv-intrinsic-rfc.md:261]: if the `vl` argument is

File: rvv-intrinsic-rfc.md [hunk 7]
```diff
@@ -246,6 +249,31 @@ size_t vsetvlmax_e8m8 ();
...
+
+The behavior is equivalence for following code, but we strongly suggesting the
```
Comment (Hsiangkai) [inline rvv-intrinsic-rfc.md:263]: The semantics of following two snippets are equivalent. We strongly suggest the first form.

**Comment by kito-cheng** [comment]
ChangeLog:
- Applied Kai's suggesion.

**Comment by kito-cheng** [comment]
ChangeLog:
- Update `rvv_c11_intrinsic_funcs.md` and `rvv_c11_intrinsic_funcs/*`

**Comment by zakk0610** [comment]
LGTM.

**Pull request merged**